templates/mobile_view_discussion_posts.mustache

```diff
@@ -14,16 +14,15 @@
 
             <ion-card-content padding-top class="card-content card-content-md">
                 <core-format-text singleline="false" text="{{CONTENT_OTHERDATA.firstpost.message}}"></core-format-text>
-
-            <!-- Uploaded files -->
-            <%#files%>
-                <core-file
-                    [file]="{fileurl: '<% fileurl %>', filename: '<% filename %>', timemodified: '<% timemodified %>', filesize: '<% filesize %>'}"
-                    component="mod_hsuforum" 
-                    componentId="<% cmid %>">
-                </core-file>
-            <%/files%>
-
+                <!-- Uploaded files -->
+                <ion-row *ngIf="CONTENT_OTHERDATA.firstpost.singleimage"> 
+                    <img style="max-height: 50vh" src={{CONTENT_OTHERDATA.firstpost.images[0].fileurl}}>
```

Comment:
Should rather apply styling with classes

Reply:
Also using a div with an background-image. Should give you more control over the image options:
div {
 background-image: url('background.jpg');
 background-size: cover;
 background-repeat: no-repeat;
}

This is just a note FYI for future use.

Reply:
sick I will add it as an update to a followup image, because the images won't size well on mobile

Reply:
removed the inline styling